spec/support/date_range_picker_shared_example.rb

```diff
@@ -17,7 +17,7 @@ def date_range_picker_select_range(range_name)
     described_class.destroy_all
   end
 
-  let!(:very_old) { create(described_class.to_s.underscore.to_sym, date_field.to_sym => Time.zone.local(1919, 7, 31)) }
+  let!(:very_old) { create(described_class.to_s.underscore.to_sym, date_field.to_sym => Time.zone.local(2000, 7, 31)) }  # We are no longer allowing dates before 2000
```

Comment:
Not sure that comment is necessary? 2000 is still "very old" :)

Reply:
addressed